services/web-server/src/loaders/tasks.js

```diff
@@ -6,16 +6,26 @@ const Task = require('../entities/Task');
 
 module.exports = ({ queue, index }) => {
   const task = new DataLoader(taskIds =>
-    Promise.all(taskIds.map(async (taskId) => {
-      try {
-        return new Task(taskId, null, await queue.task(taskId));
-      } catch (err) {
-        return err;
-      }
-    })),
+    Promise.all(
+      taskIds.map(async (taskId) => {
+        try {
+          return new Task(taskId, null, await queue.task(taskId));
+        } catch (err) {
+          return err;
+        }
+      }),
+    ),
   );
   const indexedTask = new DataLoader(indexPaths =>
-    Promise.all(indexPaths.map(indexPath => index.findTask(indexPath))),
+    Promise.all(
+      indexPaths.map(indexPath => {
+        try {
+          return index.findTask(indexPath);
```

Comment:
Hm, I'm not sure if this will throw since we're not awaiting. Perhaps this can be solved with:

```jsx
try {
  return await index.findTask(indexPath);
} catch (err) {
  return err;
}
```

Reply:
Should some other calls that aren't being awaited be awaited as well. Like the one's in `github.js` and `client.js` for example

Reply:
Good catch! Yes, please.